core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -799,8 +799,22 @@ case object ModelInsights {
       case m: RandomForestRegressionModel => Seq(m.featureImportances.toArray.toSeq)
       case m: GBTRegressionModel => Seq(m.featureImportances.toArray.toSeq)
       case m: GeneralizedLinearRegressionModel => Seq(m.coefficients.toArray.toSeq)
-      case m: XGBoostRegressionModel => Seq(m.nativeBooster.getFeatureScoreVector(featureVectorSize).toArray.toSeq)
-      case m: XGBoostClassificationModel => Seq(m.nativeBooster.getFeatureScoreVector(featureVectorSize).toArray.toSeq)
+      case m: XGBoostRegressionModel =>
+        Try(Seq(m.nativeBooster.getFeatureScoreVector(featureVectorSize).toArray.toSeq)) match {
+          case Success(contrib) => contrib
+          case _ => featureVectorSize match {
```

Comment:
Can anyone comment on why `featureVectorSize` is an Option in the first place? I couldn't find a place where this function is ever called without it being explicitly defined.

You may be able to just use `Seq(Seq.fill(n)(0.0))`, if so.

Reply:
So i can drop `case _ => Seq(Seq.empty)` with no consequences ? :D